inc/custom-header.php

```diff
@@ -8,6 +8,14 @@
  *
  * @link https://developer.wordpress.org/themes/functionality/custom-headers/
  *
+ * You can add the WooCommerce Mini Cart to header.php like so ...
+ *
+	<?php
+		if ( function_exists( '_s_woocommerce_header_cart' ) ) {
+			_s_woocommerce_header_cart();
+		}
+	?>
```

Comment:
I don't think this code is best suited here. It requires deleting code from another area if WooCommerce is not supported.